nengo_spinnaker/operators/value_sink.py

```diff
@@ -24,9 +37,10 @@ class ValueSink(object):
     sample_every : int
         Number of machine timesteps between taking samples.
     """
-    def __init__(self, probe, dt):
+    def __init__(self, probe, dt, max_width=64):
```

Comment:
Setting `max_width` lower seems to be a good idea (experimenting with larger SPA benchmarks).